index.html

```diff
@@ -2,28 +2,29 @@
 <html>
   <head>
     <title>Chat Room</title>
+    <link rel="stylesheet" href="style.css">
   </head>
 
   <body>
-    <header>
-      <h1>Chat Room</h1>
+    <header class="header">
```

Comment:
if you want to apply this style for every header in your pages, do not add class, just add it on the header element.
if you want different styles, use a more suggestive name.

do the same for nav, main and footer please